### PR TITLE
Fix /actor needlessly reading session cookie and varying on Signature

### PR DESCRIPTION
**Pull request by ClearlyClaire**

### CODE REVIEW

File: app/controllers/instance_actors_controller.rb [hunk 1]
```diff
@@ -1,8 +1,14 @@
 # frozen_string_literal: true
 
 class InstanceActorsController < ApplicationController
```
[the remark this answers is not in the document]
Comment (ClearlyClaire) [reply app/controllers/instance_actors_controller.rb:3]: Probably! Having a look.

[the remark this answers is not in the document]
Comment (ClearlyClaire) [reply app/controllers/instance_actors_controller.rb]: Done!

File: app/controllers/instance_actors_controller.rb [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+class InstanceActorsController < ActivityPub::BaseController
+  vary_by ''
 
-  skip_before_action :check_account_confirmation
-  skip_around_action :set_locale
+  serialization_scope nil
+
+  before_action :set_account
+  skip_before_action :require_functional!
+  skip_before_action :update_user_sign_in
```
Comment (Gargron) [inline app/controllers/instance_actors_controller.rb:10]: Is this line still needed? If so, do we need `update_user_sign_in` in any of the ActivityPub controllers?

Comment (ClearlyClaire) [reply app/controllers/instance_actors_controller.rb:10]: This line is needed, but no, I don't think we need it in any other of the `ActivityPub` controllers so it probably makes sense moving it to `ActivityPub::BaseController`, although I'd say it's out of scope for this PR.

**Review by Gargron — APPROVED** [review]

**Pull request merged**